newdle/migrations/versions/20250704_1127_cab5d47c1152_add_interrogation_participant.py

```diff
@@ -0,0 +1,28 @@
+"""Add interrogation participant
```

Comment:
"interrogation"?

I think something like "Ensure participant names" would be clearer... (update the filename accordingly as well)